lib/grab-project.js

```diff
@@ -14,7 +14,7 @@ function grabProject(context, next) {
     packageName = path.resolve(process.cwd(), packageName);
   }
   var bailed = false;
-  context.emit('data', 'info','npm:','Downloading project: ' + packageName);
+  context.emit('data', 'info',context.module.name,'Downloading project: ' + packageName);
```

Comment:
please include proper spaces between arguments

also please keep the info regarding npm

`context.module.name + ' npm:'`